public/plugins/testdata/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@grafana-plugins/testdata",
+  "version": "1.0.0",
+  "description": "Generates test data in different forms",
+  "private": true,
+  "main": "src/module.tsx",
+  "types": "src/module.tsx",
+  "scripts": {
+    "build": "webpack -c ./.config/webpack/webpack.config.ts --env production",
+    "dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
+    "test": "jest --watch --onlyChanged",
+    "test:ci": "jest --passWithNoTests --maxWorkers 4",
+    "typecheck": "tsc --noEmit",
+    "typecheck:showConfig": "tsc --noEmit --showConfig",
+    "lint": "eslint --cache --ignore-path ./.gitignore --ext .js,.jsx,.ts,.tsx .",
+    "lint:fix": "npm run lint -- --fix",
+    "e2e": "npm exec cypress install && npm exec grafana-e2e run",
+    "e2e:update": "npm exec cypress install && npm exec grafana-e2e run --update-screenshots",
+    "server": "docker-compose up --build",
+    "sign": "npx --yes @grafana/sign-plugin@latest"
+  },
+  "author": "Grafana",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@babel/core": "7.22.9",
+    "@grafana/e2e": "workspace:*",
+    "@grafana/e2e-selectors": "workspace:*",
+    "@grafana/eslint-config": "6.0.0",
+    "@grafana/eslint-plugin": "link:../../../packages/grafana-eslint-rules",
+    "@grafana/tsconfig": "^1.3.0-rc1",
+    "@swc/core": "1.3.38",
+    "@swc/helpers": "0.4.14",
+    "@swc/jest": "^0.2.26",
+    "@testing-library/dom": "9.3.1",
+    "@testing-library/jest-dom": "5.16.5",
+    "@testing-library/react": "14.0.0",
+    "@testing-library/user-event": "14.4.3",
+    "@types/jest": "29.2.3",
+    "@types/lodash": "4.14.195",
+    "@types/node": "18.16.16",
+    "@types/react": "18.2.15",
+    "@types/testing-library__jest-dom": "5.14.8",
+    "@typescript-eslint/eslint-plugin": "5.42.0",
+    "@typescript-eslint/parser": "5.42.0",
+    "copy-webpack-plugin": "11.0.0",
+    "css-loader": "6.8.1",
+    "eslint": "8.44.0",
+    "eslint-plugin-jsdoc": "46.4.3",
+    "eslint-plugin-react": "7.32.2",
+    "eslint-plugin-react-hooks": "4.6.0",
+    "eslint-webpack-plugin": "4.0.1",
+    "fork-ts-checker-webpack-plugin": "8.0.0",
+    "glob": "10.3.3",
+    "identity-obj-proxy": "3.0.0",
+    "jest": "29.3.1",
+    "jest-environment-jsdom": "29.3.1",
+    "prettier": "3.0.0",
+    "replace-in-file-webpack-plugin": "^1.0.6",
+    "sass": "1.63.6",
+    "sass-loader": "13.3.2",
+    "style-loader": "3.3.3",
+    "swc-loader": "^0.2.3",
+    "ts-node": "10.9.1",
+    "tsconfig-paths": "^4.2.0",
+    "typescript": "4.8.4",
+    "webpack": "5.88.1",
+    "webpack-cli": "5.1.4",
+    "webpack-livereload-plugin": "^3.0.2"
+  },
+  "engines": {
+    "node": ">= 18"
+  },
+  "dependencies": {
+    "@emotion/css": "11.11.2",
+    "@grafana/data": "10.2.0-pre",
+    "@grafana/runtime": "10.2.0-pre",
+    "@grafana/schema": "10.2.0-pre",
+    "@grafana/ui": "10.2.0-pre",
```

Comment:
should these not be workspace deps?

Reply:
these will be handled by `lerna` as the other Grafana packages: https://github.com/grafana/grafana/blob/main/packages/grafana-runtime/package.json#L40